app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,41 @@
+class RestaurantsController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
if we skip this "every time" then - do we really need it in the application controller?